composer.json

```diff
@@ -22,7 +22,7 @@
         "aws/aws-sdk-php": "~2.4|~3.0",
         "pda/pheanstalk": "~3.0",
         "php-amqplib/php-amqplib": "~2.5",
-        "phpspec/phpspec": "^2.4",
+        "phpspec/phpspec": "^4.3",
```

Comment:
This is required to bump `symfony/event-dispatcher` to version 4.x,  since `phpspec` <4.3 i s locked at 2.x or 3.x of thar lib.